Unreal/SystemLocalized/det/UnrealI.det

```diff
@@ -268,21 +268,21 @@ MenuName="Mann 2"
 
 [Sludge]
 ; EN: PickupMessage="You picked up 25 Kilos of Tarydium Biosludge"
-PickupMessage="Sie haben 25 Einheiten Tarydium-Schlamm aufgenommen"
+PickupMessage="25 Einheiten Tarydium-Bioschlamm erhalten"
 ; EN: ItemName="Tarydium Biosludge"
-ItemName="Tarydium-Schlamm"
+ItemName="Tarydium-Bioschlamm"
 
 [Invisibility]
 ; EN: ExpireMessage="Invisibility has worn off."
-ExpireMessage="Die Tarnung hat verbraucht."
+ExpireMessage="Die Unsichtbarkeit wurde verbraucht."
 ; EN: PickupMessage="You have Invisibility"
-PickupMessage="Du hast Tarnung"
+PickupMessage="Unsichtbarkeit erhalten"
 ; EN: ItemName="Invisibility"
-ItemName="Tarnung"
+ItemName="Unsichtbarkeit"
```

Comment:
@SlyMk3 I made it "Unsichtbarkeit" as the original name could have used the same logic you are proposing here, but specifically went with "Invisibility" rather than "Stealth Cloak" or "Cloaking Device" or anything along those lines.

Reply:
Besides, UPak has a Cloaking Device item. So it's important to distinguish the two.